scripts/combine_lightcurves.py

```diff
@@ -1,43 +1,93 @@
+#!/usr/bin/env python3
+
 import os
+import shutil
 import json
 import argparse
+from json.decoder import JSONDecodeError
 
-import glob
+from pocs.utils.google.storage import PanStorage
 
 
 def get_curves_for_pic(pic):
+    """Read light curve files and add all curves to array.
+
+    :param pic: the PIC id for the star to make a master light curve for
+    :return: an array of all the light curves for the given PIC
+    """
     curves = []
-    for filename in glob.iglob("data/LC/{}/**/*.json".format(pic), recursive=True):
-        with open(filename, 'r') as f:
-            c = json.load(f)
-            curves.append(c)
-    return curves
+    bucket_name = 'panoptes-simulated-data'
+    prefix = "LC/{}/".format(pic)
+    temp_dir = '{}/temp'.format(os.getenv('PANDIR', default='/var/panoptes'))
+
+    pan_storage = PanStorage(bucket=bucket_name)
+    blobs = pan_storage.list_remote(prefix)
+
+    for blob in blobs:
+        local_path = "{}/{}".format(temp_dir, blob.name)
+        path = pan_storage.download(blob.name, local_path=local_path)
+        with open(path, 'r') as f:
+            try:
+                curve = json.load(f)
+                curves.append(curve)
+            except JSONDecodeError:
+                print("Error: Object could not be decoded as JSON.")
+    return curves, temp_dir
```

Comment:
Do we need to return the `temp` dir here? I'd almost rather see an optional param passed into this function that let's you specify the download destination otherwise the temp dir.